### ISSUE TITLE
Improve `Naming/PredicateName` cop

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#12358]
## Description

Avoid prefixing predicate methods with the auxiliary verbs such as is, does, or can. These words are redundant and inconsistent with the style of boolean methods in the Ruby core library, such as `empty?` and `include?`.

https://rubystyle.guide/#bool-methods-prefix

## Describe the solution you'd like

```ruby
# bad
class Person
  def can_play_basketball?
    false
  end

  def does_like_candy?
    true
  end
end

# good
class Person
  def basketball_player?
    false
  end

  def likes_candy?
    true
  end
end
```

**Comment by fatkodima** [comment]
This can already be configured via `NamePrefix` option. Probably, we should extend that list with these values by default.
